packages/ui/src/solidity/inject-hyperlinks.ts

```diff
@@ -2,17 +2,32 @@ import { version as contractsVersion } from '@openzeppelin/contracts/package.jso
 
 export function injectHyperlinks(code: string) {
   // We are modifying HTML, so use HTML escaped chars. The pattern excludes paths that include /../ in the URL.
-  const contractsRegex =
+  const importContractsRegex =
     /&quot;(@openzeppelin\/)(contracts-upgradeable\/|contracts\/)((?:(?!\.\.)[^/]+\/)*?[^/]*?)&quot;/g;
-  const communityContractsRegex = /&quot;(@openzeppelin\/)(community-contracts\/)((?:(?!\.\.)[^/]+\/)*?[^/]*?)&quot;/g;
+  const importCommunityContractsRegex =
+    /&quot;(@openzeppelin\/)(community-contracts\/)((?:(?!\.\.)[^/]+\/)*?[^/]*?)&quot;/g;
 
-  return code
-    .replace(
-      contractsRegex,
-      `&quot;<a class="import-link" href="https://github.com/OpenZeppelin/openzeppelin-$2blob/v${contractsVersion}/contracts/$3" target="_blank" rel="noopener noreferrer">$1$2$3</a>&quot;`,
-    )
-    .replace(
-      communityContractsRegex,
-      `&quot;<a class="import-link" href="https://github.com/OpenZeppelin/openzeppelin-community-contracts/blob/master/contracts/$3" target="_blank" rel="noopener noreferrer">$1$2$3</a>&quot;`,
-    );
+  const compatibleCommunityContractsRegexSingle = /OpenZeppelin Community Contracts commit ([a-f0-9]{40})/;
```

Comment:
Accordingly, if we use a single line

```suggestion
  const compatibleCommunityContractsRegexSingle = /Community Contracts commit ([a-f0-9]{40})/;
```